5FNSaaS/SamsungAutomationStudio
# Request to add motion pose nodes (#46)

* refactor: directory 구조 수정 및 node명 변경
* code: hand-pose.js
* update hands detection socket html
* update hands detection node paletteLabel
* model sample html
* fix: weigtedDastance edit
* modify: input media detection page 변경
* update timer setting and motion name setting
* feat: offscreen - 탭 비활성화 되어도 작동가능
* comment: 주석 추가
* add: html timer
* bug: start canvas on video loaded event
* add: 포즈 등록(타이머 X)
* feat: motion detection result deatil table display
* test: blob via websocket
* blob
* style: hands motion detail table comment
* feat: hand motion name and regist pf value send
* add: flow until saving keypoint data in mongodb
* add: motion registration timer setting
* feat: hand-similarity node-red setting
* feat: Create pose-similarity-register node
* fix: modify node names
* feat: input data exception
* feat: add hand-find node
* feat: websocket html update
* add: captured pose table
* feat: monitoring node
* refactor
* feat: Apply accumulated keypoints
* feat: Apply palletteLabel
* feat: modify hand similarity logic
* feat: dependencies setting
* Create README.md
* modify: change format of transmitted pose data
* style: add comments to html
* docs: update first README.md
* finish: 주석 추가 및 코드 리팩토링 완료
* update: flow 업데이트
* feat: same name check
* Add files via upload
feat: add name check
* docs: git ignore
* merge: 이진과 머지
* update: socket.io close 이벤트 추가
* merge: detection hogeun repo
* merge: similarity yeorae repo
* merge: detection ijin repo
* feat: check same input data
* merge: similarity hyeonmin repo
* merge: detection hyeona repo
* refactor: integrate each node developments
* fix: pose-similarity node Return msg update
* fix: pose-similarity-register output update
* fix: modify visibility error
* add: pose registration trigger from socket server
* style: regist btn and copyright & favicon update
* feat: error case message and regist btn function
* feat: hands detection socket html update
* feat: hand detection flow create
* add: regist btn and copyright
* docs: readme update
* feat: rtsp streaming & body detection
* feat: hands detection update data merge
* docs: create gitignore about node and vscode
* Create README.md
* feat: hand-detection html
* feat: hand-detection js
* feat: hand-find html
* feat: hand-find js
* feat: hand-register html
* feat: hand-register js
* feat: node package json
* Create README.md
* style: merge repo detele
* docs: update readme - git repository url
* Add files via upload
docs: hand pose find-regist flow
* fix: registerType name modify
* refactor: monitor node
* refactor: mediapipe body pose
* feat: Node return edit
* feat: Add yearaeflow (Pose control IoT Device Proto)
* refactor: code refactor
* modify: capture parameter
* Update README.md
* feat: 노드 플로우 상 360 카메라 노드에서 url 직접 받도록 수정
* feat: autoplay
* fix: modify socket url
* update: html 문서에 rtsp url 입력창 제거
* fix: timer data bug solved
* fix: timer send data update
* feat: custom input for ws/
* undo: bixby socket onmessage
* debug: poseData[poseName] = motionName
* update: custom input for 'ws/'
* fix: websocket parameter name
* undo: bixby socket onmessage
* update: monitor part for mediapipe node
* update: monitor
* update: monitor url
* test: log
* feat: Similar Sensitivity input edit
* test: cors
* test: /ws/monitor
* feat: custom input for socket url and port
* docs: add README.md
* refactor: add new directory for integrated version
* update: custom input for socket url and port
* feat: html file for recognize postures
* refactor: pose-detection nodes
* delete: pose-detection-view.html
* update: package.json for body nodes
* debug: 오타 수정
* update: iot camera description
* feat: html file for find similar postures
* debug: body pose node iotcam
* html file for register new postures
* update: package.json
* feat: iotcam node
* update: iotcam node
* style: Rearrange variable def
* gitignore
* style: Rearrange variable
* fix: Edit pose-similarity nodes input and output
* etc: pose-similarity-yeorae Folder Update
* docs: Make pose similarity nodes Readme
* docs: add LICENSE
* fix: html edit
* fix: node's registerType mismatch
* feat: add help to pose-similarity-find node
* refactor: modify input property name
* feat: add help to pose-similarity-register node
* feat: add help to pose-detection-webcam node
* feat: add help to pose-detection-iotcam node
* feat: add help to monitor node
* chore: modify directory structure and package.json
* docs: add .gitignore
* docs: update .gitignore
* feat: add help to hand-find node
* feat: add help to hand-register node
* chore: delete additional files
* docs: pose-detection README
* docs: pose nodes README
* docs: monitor node README
* docs: hands detection nodes README
* docs: hands find node README
* docs: hands find and register nodes README
* docs: update main README
* docs: hands find and register nodes README
* docs: modify pose-similarity-find node, pose-similarity-register node README
* docs: modify json example of pose detection nodes
* docs: hands detection nodes README
* docs: update README hogeun
* update: MNID/PAT property of iotcam nodes
* update: add tips for MNID/PAT property on iotcam nodes
* style: package json description anc author update
* releases: version update
* docs: update README, pre-requisites of iotcam nodes
* docs: node info update and wiki
* docs: reference update
* style: delete unnecessary comments
* release: version update
* feat: pose detection iotcam node
* feat: pose detection webcam node
* feat: pose similarity find node
* feat: pose similarity register node
* feat: hand detection iotcam node
* feat: hand detection webcam node
* feat: hand find node
* feat: hand register node
* feat: motion pose monitor node
* feat: body home training flow
* feat: body iot control flow
* feat: hand iot control flow
* feat: motion pose node and dependencies update
* refactor: separate html code from node module
* style: remove comments
* refactor: separate html code from pose-detection-webcam node module
* update: credential prop for iotcam nodes
* refactor: separate html code from pose-detection-iotcam node module
* update: dependency setup (node-rtsp-stream/mpeg1muxer.js)
* refactor: separate html code from monitor node module
* refactor: separate html code from hand-detection-iotcam node module
* refactor: separate html code from hand-detection-webcam node module
* refactor: modify library require
* refactor: modify html file naming
* update: construct socket when the flow is deployed - monitor.js
* refactor: modify library require
* style: restore origin comments
* chore: edit logs
* update: move 'require()' to the top of the code
* release: version update
* rename: detection -> detect
* rename: similarity-find -> find / similarity-register -> register
* chore: correct mistype
* docs: update README
* update: help of iotcam nodes
* release: version update
* refactor: clean up code that reflects feedback

Co-authored-by: 윤이진 <483759@naver.com>
Co-authored-by: 조현민 <hyeonmean@gmail.com>
Co-authored-by: steven9408 <whdufo@gmail.com>
Co-authored-by: Root_Kwak <rootkwak528@gmail.com>
Co-authored-by: hmiin <57279882+whgusalsdl@users.noreply.github.com>
